### ISSUE TITLE
Allow Multilanguage MotD/Donor Page

### ISSUE THREAD

**Issue by bbdoc** [WatWowMap/ReactMap#281]
Would it be possible to allow multilanguages for the MotD and Donor Pages ?

The way I see it would be to add an optional `language` field to every component, e.g. : 

```
        {
          "type": "text",
          "content":"This text will show for every language",
        },
        {
          "type": "text",
          "language": "EN",
          "content":"This text will only show if user language is defined as English",
        },
        {
          "type": "text",
          "language": "FR",
          "content":"This text will only show if user language is defined as French",
        },
```

Thanks

**Comment by Petap0w** [comment]
+1, although maybe better to make `content` an array of objects, one for each locale, so we don't have to repeat all the component block (gridSizes, style, etc.).
Same for button `name`

**Comment by TurtIeSocks** [comment]
Resolved in #451
